lib/action_policy/lookup_chain.rb

```diff
@@ -66,22 +65,17 @@ def call(record, **opts)
       private
 
       def lookup_within_namespace(policy_name, namespace, strict: false)
-        return unless namespace
-        NamespaceCache.fetch(namespace.name, policy_name, strict: strict) do
+        NamespaceCache.fetch(namespace&.name || :__global__, policy_name, strict: strict) do
```

Comment:
I think, we can use "Kernel" as the global namespace

```suggestion
        NamespaceCache.fetch(namespace&.name || "Kernel", policy_name, strict: strict) do
```